services/validators_test.go

```diff
@@ -163,12 +167,15 @@ func TestValidateServiceAgreement(t *testing.T) {
 		{"less than minimum payment", basic.Add("payment", "1"), true},
 		{"less than minimum expiration", basic.Add("expiration", 1), true},
 		{"without being listed as an oracle", basic.Add("oracles", []string{}), true},
+		// XXX: This is not being properly tested ATM, because the actual config.MaximumServiceDuration value is 0!!!
+		{"past allowed end at", basic.Add("endAt", "3000-06-19T22:17:19Z"), true},
+		// {"before allowed end at", basic.Add("endAt", "2018-06-19T22:17:19Z"), true},
```

Comment:
Likewise:

---
We try not to leave commented out code when we merge to master. Should we make an issue for this in pivotal tracker?

Reply:
Thanks again for catching these. Fixed in 84308154046efc125388.